system.prop

```diff
@@ -1,6 +1,40 @@
-#
```

Comment:
I understand what Shr3ps is saying. It's only the first comment that he wants to keep in order to grep in into the build.prop file once generated on the target.

could you just readd `# system.prop for A6020` on the first line. Just for debugging purposes. I'll take the PR right after.

Thanks a lot for all this, I'm busy right now and don't have time to work on the device.